### PR TITLE
w: add -i example, uniq: additions, seq: add page

### PR DESCRIPTION
**Pull request by notpeter**
Squashed, rebased version of #457 which fixes the outstanding merge conflicts and drops the edits to `ls` because they were shell globbing examples.

### CODE REVIEW

**Comment by notpeter** [comment]
I also added the punctuation which caused test to fail, squashed and repushed.

**Comment by raamana** [comment]
thanks.

**Comment by rubenvereecken** [comment]
You can _do_ this? Multiple authors? What is this magick.

File: pages/common/w.md [hunk 1]
```diff
@@ -10,3 +10,7 @@
 - Show logged-in users info without a header:
 
 `w -h`
+
+- Show info about logged-in users, sorted their idle time:
```
Comment (rubenvereecken) [inline pages/common/w.md]: sorted _by_

**Comment by igorshubovych** [comment]
`-i` has different meaning for OSX and Linux versions

**Comment by notpeter** [comment]
Put `w -i` into new osx/w.md and changed it to "sorted by", left common/w.md unchanged.

**Comment by igorshubovych** [comment]
it is actually interesting. should we move page from common to linux?

**Comment by notpeter** [comment]
The old example truly is "common" I double checked and Linux/Solaris both support those same limited flags so I think we should leave it as is.

File: pages/common/uniq.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # uniq
 
-> Report or omit repeated lines.
+> Output the unique lines from the given input or file.
+> Compare the adjacent files in input (file or standard input) and outputs unique lines.
```
Comment (igorshubovych) [inline pages/common/uniq.md]: Does it really makes sense to have such long description?
I like the original description or just a first line of your description. I think that 2nd line of your description does not add too much and hence I'd rather remove it.

Comment (raamana) [inline pages/common/uniq.md]: I agree with @igorshubovych

**Comment by raamana** [comment]
Great work and discussion guys., Lets get this merged already. Lets not defeat the purpose of this project with too long a discussion..

**Comment by rubenvereecken** [comment]
@raamana we've got an agreement among contributors to only merge after two :+1:s, which can create some delays in times of less activity. Anyway, here goes mine after @igorshubovych's concerns are addressed: :+1:

File: pages/common/uniq.md (hunk 2 above)
Comment (waldyrious) [inline pages/common/uniq.md]: :+1: for using the first line of the new description.

Comment (notpeter) [inline pages/common/uniq.md]: Done.

**Comment by rubenvereecken** [comment]
Et voila, finalement :tada: 

Thanks @notpeter!

**Pull request merged**